vx/scripts/release/steps/publishPackage.ts

```
@@ -5,33 +5,71 @@ import * as taggedBranch from 'vx/util/taggedBranch.js';
     throw new Error('Package context is required for publishing.');
   }
 
+  // High-signal logging so CI output clearly shows what we intend to publish.
   logger.info(`🚀 Publishing package ${usePackage()}.
     Version: ${versionToPublish}
     Tag Id: ${tagId}
     Tag: ${tag}`);
 
+  // Respect branch policy. If publishing is not allowed, exit this step early.
   if (!shouldRelease(versionToPublish)) {
     return logger.info(`❌  Not in release branch. Skipping publish.`);
```

Biome (2.4.6)

[error] The function should not return a value because its return type is void.

(lint/correctness/noVoidTypeReturn)